plugins/modules/azure_rm_resource_info.py

```diff
@@ -98,6 +104,14 @@
   azure_rm_resource_info:
     resource_group: "{{ resource_group }}"
     resource_type: resources
+
+- name: Get all snapshots of all resource groups of a subscription but filtering with two tags. 
```

Comment:
Delete trailing space
```suggestion
- name: Get all snapshots of all resource groups of a subscription but filtering with two tags.
```